sdk/textanalytics/Azure.AI.TextAnalytics/src/SentenceAspect.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core;
+
+namespace Azure.AI.TextAnalytics.Models
+{
+    [CodeGenModel("SentenceAspect")]
+    internal partial class SentenceAspect
+    {
+        public string Sentiment { get; }
```

Comment:
Transform from enum `SentenceAspectSentiment` to string so we can parse it into a `TextSentiment`

Reply:
Nit - Consider adding a comment in the source directing readers to the transform?